src/ring_theory/algebraic_independent.lean

```diff
@@ -0,0 +1,433 @@
+/-
+Copyright (c) 2021 Chris Hughes. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Chris Hughes
+-/
+import ring_theory.adjoin.basic
+import linear_algebra.linear_independent
+import ring_theory.mv_polynomial.basic
+import data.mv_polynomial.supported
+import ring_theory.algebraic
+import data.mv_polynomial.equiv
+/-!
+# Algebraic Independence
+
+This file defines algebraic independence of a family of element of an `R` algebra
+
+## Main definitions
+
+* `algebraic_independent` - `algebraic_independent R v` states the family of vectors `v`
```

Comment:
I wouldn't call elements of a random `R`-algebra "vectors". I think just calling the "elements" is more in line with common use in the literature.
Indeed, the Stacks reference that you link to also talks about a "collection of elements". I would probably also use the notation `x` or `a` for this family of elements.